### ISSUE TITLE
check_dhtxx missing GPIO info

### ISSUE THREAD

**Issue by 76bugguy01** [NEMSLinux/nems-plugins#11]
when check_dhtxx is used NEMS returns error:  (Service check timed out after 120.09 seconds)

Command executed:  /usr/lib/nagios/plugins/check_dhtxx 35 40 75 80 temp

Looking at check_dhtxx I see lines 13 and 15 are calling nems-info but are failing to pass a second argument which specify which GPIO pin is in use.  nems-info doesn't have a default value for the GPIO pin so the command eventually times out.

line 13:  $dht = json_decode(shell_exec('/usr/local/bin/nems-info dht22'));

needs to be changed to: $dht = json_decode(shell_exec('/usr/local/bin/nems-info dht22 18'));  

or ideally NConf should prompt for GPIO pin and pass it as an argument to check_dhtxx.

**Comment by NEMSLinux** [comment]
Good catch - thank you. This is due to the changes encountered in some of the newer DHT modules that require a different pin be used. I [added the capability](https://github.com/NEMSLinux/nems-scripts/blob/main/debpack/usr/local/share/nems/nems-scripts/dhtxx#L18-L34) but missed updating the check command itself (since the default is pin 4 which works with my test device). I will roll this fix dynamically into the check command in an upcoming update (after 1.6.1 is released). Thanks again!

**Comment by 76bugguy01** [comment]
Looking a bit closer, I found this issue because I have a KY-015 version of DHT11 sensor and NEMS docs for that state to connect to GPIO pin 18.

https://docs.nemslinux.com/en/latest/accessories/dhtsensors.html#ky-015-dht11-sensor-on-3-pin-pcb
